### PR TITLE
kernel: bump 5.4 to 5.4.162

### PR DESCRIPTION
**Pull request by graysky2**
All patches automatically rebased.
```
Build system: x86_64
Build-tested: ramips/mt7621*
```
*I am hit with the binutils 2.37 bug so I had to revert 7f1edbd41295dff9f2127b169fbc086c0fb2c14e in order to downgrade to 2.35.1

Signed-off-by: John Audia <graysky@archlinux.us>

### CODE REVIEW

File: target/linux/layerscape/patches-5.4/804-crypto-0016-MLKU-114-1-crypto-caam-reduce-page-0-regs-access-to-.patch [hunk 1]
```diff
@@ -100,19 +100,20 @@ Signed-off-by: Horia Geantă <horia.geanta@nxp.com>
...
+ 	if (priv->era < 10) {
 -		pk_inst = (rd_reg32(&priv->ctrl->perfmon.cha_num_ls) &
 +		pk_inst = (rd_reg32(&priv->jr[0]->perfmon.cha_num_ls) &
  			   CHA_ID_LS_PK_MASK) >> CHA_ID_LS_PK_SHIFT;
- 	else
--		pk_inst = rd_reg32(&priv->ctrl->vreg.pkha) & CHA_VER_NUM_MASK;
+ 	} else {
+ 		pkha = rd_reg32(&priv->ctrl->vreg.pkha);
+-		pk_inst = pkha & CHA_VER_NUM_MASK;
 +		pk_inst = rd_reg32(&priv->jr[0]->vreg.pkha) & CHA_VER_NUM_MASK;
```
Comment (hauke) [inline target/linux/layerscape/patches-5.4/804-crypto-0016-MLKU-114-1-crypto-caam-reduce-page-0-regs-access-to-.patch:113]: This change looks wrong to me, I think this part should look like this:
```
@@ -1099,10 +1099,10 @@ int caam_pkc_init(struct device *ctrldev
 
        /* Determine public key hardware accelerator presence. */
        if (priv->era < 10) {
-               pk_inst = (rd_reg32(&priv->ctrl->perfmon.cha_num_ls) &
+               pk_inst = (rd_reg32(&priv->jr[0]->perfmon.cha_num_ls) &
                           CHA_ID_LS_PK_MASK) >> CHA_ID_LS_PK_SHIFT;
        } else {
-               pkha = rd_reg32(&priv->ctrl->vreg.pkha);
+               pkha = rd_reg32(&priv->jr[0]->vreg.pkha);
                pk_inst = pkha & CHA_VER_NUM_MASK;
 
                /*
```

**Review by hauke — COMMENTED** [review]
Please add the add the Signed-off-by line after the
```
Build system: x86_64
Build-tested: ipq40xx/EX6100 v2*, ramips/mt7621*
```

File: target/linux/layerscape/patches-5.4/804-crypto-0016-MLKU-114-1-crypto-caam-reduce-page-0-regs-access-to-.patch (hunk 1 above)
Comment (graysky2) [reply target/linux/layerscape/patches-5.4/804-crypto-0016-MLKU-114-1-crypto-caam-reduce-page-0-regs-access-to-.patch]: Fixed and rebased.

**Comment by graysky2** [comment]
At the time I rebased this lantiq was on 5.4 but after I push the commit, lantiq went to 5.10.  I resolved a few conflicts.  Do you want that captured?

File: target/linux/lantiq/patches-5.4/0001-MIPS-lantiq-add-pcie-driver.patch [hunk 2]
```diff
@@ -0,0 +1,5506 @@
+From 6f933347d0b4ed02d9534f5fa07f7b99f13eeaa1 Mon Sep 17 00:00:00 2001
```
Comment (abajk) [inline target/linux/lantiq/patches-5.4/0001-MIPS-lantiq-add-pcie-driver.patch:1]: Something went wrong during the rebassing. This file should be deleted.

**Review by abajk — CHANGES_REQUESTED** [review]

File: target/linux/lantiq/patches-5.4/0001-MIPS-lantiq-add-pcie-driver.patch (hunk 2 above)
Comment (graysky2) [reply target/linux/lantiq/patches-5.4/0001-MIPS-lantiq-add-pcie-driver.patch]: Yes, I believe github threw two conflicts, that file and also `target/linux/lantiq/patches-5.4/0152-lantiq-VPE.patch`

I just deleted `target/linux/lantiq/patches-5.4/0001-MIPS-lantiq-add-pcie-driver.patch` .. shal I do the same for the other?

Comment (abajk) [reply target/linux/lantiq/patches-5.4/0001-MIPS-lantiq-add-pcie-driver.patch]: You're right. Both files should be deleted.

Comment (graysky2) [reply target/linux/lantiq/patches-5.4/0001-MIPS-lantiq-add-pcie-driver.patch]: Done.

**Comment by hauke** [comment]
kernel 5.4 support for lantiq was removed some hours ago. Please rebase on top of current master and do not add these patches again.

**Comment by graysky2** [comment]
Done.

**Comment by hauke** [comment]
Thank you for the patch, I applied it with little modifications to master in bbdc13b15b383b8b8d5492575e0a83cbe2af139b and 81995a5e77c0cc5bf5b59ef4bec78b969c020013

**Pull request closed without merging**